### PR TITLE
Pronoun Support based on #69

### PR DESCRIPTION
**Pull request by vatsalyagoel**
image

### CODE REVIEW

File: src/Form/Form.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ export class Form extends Component {
   static labels = {
     signatureTypes: { label: 'Signature Type', type: 'radio' },
     name: { label: 'Your Name', required: true },
+    pronoun: { label: 'Your Pronouns', required: true },
```
Comment (robdmoore) [inline src/Form/Form.js:16]: Should we make it required or opt-in? cc @kjacobsen

File: src/Form/__snapshots__/Form.test.js.snap [hunk 2]
```diff
@@ -123,6 +123,53 @@ exports[`Renders 1`] = `
...
+    >
+      <div
+        className="control"
+        key="pronoun"
+      >
+        <input
+          className="input"
+          id="input-pronoun"
+          onChange={[Function]}
+          placeholder="he/him | she/her | they/them"
```
Comment (robdmoore) [inline src/Form/__snapshots__/Form.test.js.snap:161]: nice!

File: src/Form/Form.js (hunk 1 above)
Comment (kjacobsen) [reply src/Form/Form.js:16]: Part of me wants to make it mandatory to encourage everyone to list their pronouns. It normalises the practice for everyone.

Comment (kjacobsen) [reply src/Form/Form.js:16]: We should probably make this optional.

@vatsalyagoel Do we just need to change the required here, or will we need to do updates in other places?

Comment (vatsalyagoel) [reply src/Form/Form.js:16]: The formatting would have to remove the brackets in the output but we should be able to do it

Comment (vatsalyagoel) [reply src/Form/Form.js]: @robdmoore Made Pronoun optional

**Pull request merged**